packages/cli/project.json

```diff
@@ -2,7 +2,7 @@
   "name": "cli",
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
   "sourceRoot": "packages/cli/src",
-  "projectType": "application",
+  "projectType": "library",
```

Comment:
cli and create-* packages are of project type library. This is due to the nature of how they are published, as package. In comparison we have applications that are deployed.